app/assets/v2/css/activity_stream.css

```diff
@@ -5,26 +5,33 @@
   background-color: #ecf0f4;
```

Comment:
In general, please do not include arbitrary style changes in PRs as it makes it difficult to do a code review. Please limit your changes to the feature or fix thats described in the PR description - if you want to do a style fix PR, you can submit it separately.

Reply:
I've removed the code style changes and left only changes I've made for the feature. I will follow this rule of thumb from now onwards.